repo/listing.go

```diff
@@ -1057,16 +1057,27 @@ func (l *Listing) GetOptions() ([]*pb.Listing_Item_Option, error) {
 // GetSkus - return item skus
 func (l *Listing) GetSkus() ([]*pb.Listing_Item_Sku, error) {
 	var (
-		sl  = &pb.SignedListing{}
-		err = json.Unmarshal(l.ListingBytes, &sl)
+		pbl = &pb.Listing{}
+		err = jsonpb.UnmarshalString(string(l.ListingBytes), pbl)
```

Comment:
Does this occur in other places?

Reply:
It looks like the other places it's appropriate to use regular JSON.